src/components/filter-sidebar/filter-sidebar.js

```diff
@@ -1,8 +1,19 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
 import { Sidebar, Box, Label } from 'grommet';
 import FilterSidebarForm from './filter-sidebar_form';
+import { sidebarEnabled } from '../../actions/sidebar';
 
 class FilterSidebar extends React.Component {
+  constructor() {
+    super();
+    this.onClose = this.onClose.bind(this);
```

Comment:
I suggest using public class fields syntax:
https://reactjs.org/docs/handling-events.html

It's enabled in create-react-app by default

Reply:
Should  I use **Experimental** public class fields syntax?

Reply:
yes)